units/getty@.service.m4

```diff
@@ -46,10 +46,6 @@ KillMode=process
 IgnoreSIGPIPE=no
 SendSIGHUP=yes
 
-# Unset locale for the console getty since the console has problems
-# displaying some internationalized messages.
-Environment=LANG= LANGUAGE= LC_CTYPE= LC_NUMERIC= LC_TIME= LC_COLLATE= LC_MONETARY= LC_MESSAGES= LC_PAPER= LC_NAME= LC_ADDRESS= LC_TELEPHONE= LC_MEASUREMENT= LC_IDENTIFICATION=
```

Comment:
thinking about  this, i am not sure how i feel about this... the problem is that the console supports a tiny subset of utf8 only (as it can only cover 512 glyphs at a time). 

if you want proper language support, don't use the console, it's that easy really.

Also see 1640944a847249d3f5f0fb0d5a5f820a82efaed0.

To my knowledge little has changed since 2011 about support for non-european glyphs really? dunno? the console code is pretty much unmainted in the kernel afaics, and nobody is interested in reworking it..

I think we should inherit env vars everywhere properly, but I am not sure that for console logins this is really the best option.

So, I am tempted to say we should apply the env var inhreiting part of it, but I am tempted to still say we should reset all language related bits of it on console logins. unless something really changed in that regard but I think it didn't.

I figure @spbnick's goal with this is kinda colliding with this, not sure what to do?